e2e/__tests__/__snapshots__/failures.test.ts.snap

```diff
@@ -211,7 +211,7 @@ FAIL __tests__/duringTests.test.js
       35 | 
       36 | test('done(non-error)', done => {
 
-      at Object.<anonymous>.done (__tests__/duringTests.test.js:33:8)
+      at Object.<anonymous> (__tests__/duringTests.test.js:33:8)
```

Comment:
I have no idea why this has changed...

Reply:
hmmmmm, seems to just happen on node 12... Seeing as we don't test on node 12, I just reverted it 💪